src/__tests__/unit/operate.test.js

```diff
@@ -0,0 +1,24 @@
+import operate from '../../logic/operate';
+
+describe('Validate and run operands test functionality', () => {
+  test('multiply', () => {
+    const result = operate('1', '2', 'x');
+    expect(result).toBe('2');
+  });
+  test('add', () => {
+    const result = operate('1', '2', '+');
+    expect(result).toBe('3');
+  });
+  test('modular', () => {
+    const result = operate('2', '2', '%');
+    expect(result).toBe('0');
+  });
+  test('divide', () => {
```

Comment:
- [ ] As requested by the previous code reviewer, please kindly add a test for division by zero.

Reply:
Hello @Mihndim2020 , This was not requested by the previous reviewer. Nevertheless, I'll do. Hopefully, the next reviewer does not suggest the changes you have not pointed out.

Thanks a lot && HappyCoding 👨🏽‍💻

Reply:
Actioned.